app/src/mixins/search-mixin.ts

```diff
@@ -105,7 +105,10 @@ export class SearchMixin extends Mixins(CommonMixin) {
     return (
       corpType === CorpTypeCd.BC_COMPANY ||
       corpType === CorpTypeCd.BENEFIT_COMPANY ||
-      corpType === CorpTypeCd.BC_ULC_COMPANY
+      corpType === CorpTypeCd.BC_ULC_COMPANY ||
+      corpType === CorpTypeCd.CONTINUE_IN ||
+      corpType === CorpTypeCd.BEN_CONTINUE_IN ||
+      corpType === CorpTypeCd.ULC_CONTINUE_IN
```

Comment:
And this specifically excludes alterations "from a CCC to something else", right?

Reply:
yup! you can verify by testing with **C9900834**

https://namerequest-dev--pr-782-jvlgs0r3.web.app/